collect_app/src/main/java/org/odk/collect/android/instancemanagement/InstanceListItemView.kt

```diff
@@ -0,0 +1,122 @@
+package org.odk.collect.android.instancemanagement
+
+import android.content.Context
+import android.view.View
+import android.widget.ImageView
+import android.widget.TextView
+import org.odk.collect.android.R
+import org.odk.collect.android.utilities.FormsRepositoryProvider
+import org.odk.collect.forms.instances.Instance
+import org.odk.collect.strings.R.string
+import timber.log.Timber
+import java.text.SimpleDateFormat
+import java.util.Date
+import java.util.Locale
+
+object InstanceListItemView {
+
+    @JvmStatic
+    fun setInstance(view: View, instance: Instance, shouldCheckDisabled: Boolean) {
+        val context = view.context
+
+        val imageView = view.findViewById<ImageView>(R.id.image)
+        setImageFromStatus(imageView, instance)
+        setUpSubtext(view, instance, context)
+
+        val chip = view.findViewById<TextView>(R.id.chip)
+        if (chip != null) {
```

Comment:
Is it possible that it's null?

Reply:
Yeah because the adapter sometimes uses a different layout (`form_chooser_list_item_multiple_choice.xml`).